### PR TITLE
Milestone 28 vsdk 4.1.0

### PR DESCRIPTION
**Pull request by billy-the-fish**

### CODE REVIEW

File: shared/extensions-marketplace/virtual-background/project-implementation/android.mdx [hunk 1]
```diff
@@ -0,0 +1,84 @@
+<PlatformWrapper platform='android'>
+
+This section explains how to enable your users to choose a virtual background.
+
+### Implement the user interface
+
+To enable and change virtual backgrounds, you add a button the user interface. To do this, open `/app/res/layout/activity_main.xml` and add the following lines before `</RelativeLayout>`:
```
Comment (atovpeko) [inline shared/extensions-marketplace/virtual-background/project-implementation/android.mdx:7]: a button TO the user interface? (other platforms, too)

File: shared/video-sdk/develop/product-workflow/project-implementation/windows.mdx [hunk 2]
```diff
@@ -0,0 +1,186 @@
+<PlatformWrapper platform = "windows">
+
+
+### Implement the user interface
+
+In a real-world application, for each volume setting you want to control, you typically add a volume control UI element such as a <Link target="_blank" to="https://learn.microsoft.com/en-us/cpp/mfc/using-csliderctrl?view=msvc-170">Slider Control</Link> to the audio configuration panel. To enable the user to mute local or remote audio, you add a switch or a Check Box to the interface for each user. In this example, you add a `Slider Control` and a `Check Box` to the UI to test different volume settings. For screen sharing, you add a `Button` to start and stop sharing. To
+Implement this UI, take the following steps:
```
Comment (atovpeko) [inline shared/video-sdk/develop/product-workflow/project-implementation/windows.mdx:7]: Implement --> implement

**Review by atovpeko — APPROVED** [review]
Approved with tiny comments

File: shared/video-sdk/develop/product-workflow/project-implementation/windows.mdx (hunk 2 above)
Comment (billy-the-fish) [reply shared/video-sdk/develop/product-workflow/project-implementation/windows.mdx]: Done

**Pull request merged**